phase2/addon/lib/utils.js

```diff
@@ -416,6 +416,19 @@ exports.overridePrefs = function(fileName){
   }
 }
 
+exports.findButtonPlacement = function(name){
+  let placements = JSON.parse(require("sdk/preferences/service")
```

Comment:
if we were fancy, this should by `try` `catch`, because it can be an unset pref, maybe?